.github/workflows/cd.yml

```diff
@@ -11,22 +11,29 @@ on:
 jobs:
   check_release:
     runs-on: ubuntu-22.04
+    outputs:
+      should_run: ${{ steps.check.outputs.should_run }}
     steps:
       - name: Check if release merge or workflow dispatch
-        id: check_release
+        id: check
```

Comment:
Do we need to set an `id:` here?

If this is needed for outputs, then this line below probably needs to be renamed too:
```
if: needs.check_release.outputs.should_run == 'true'
```